lib/Onyx.js

```diff
@@ -6,8 +6,8 @@ import Storage from './storage';
 import * as Logger from './Logger';
 import cache from './OnyxCache';
 import createDeferredTask from './createDeferredTask';
-import mergeWithCustomized from './mergeWithCustomized';
-import { merge } from './fastMerge';
+// eslint-disable-next-line rulesdir/prefer-import-module-contents
+import {fastMerge} from './fastMerge';
```

Comment:
By using a default export in `fastMerge.js` that will clean this up and make the ESLint warning go away.

```suggestion
import fastMerge from './fastMerge';
```

Reply:
That's true but it's a bad practise in my opinion to use default exports. How can you find all occurrences of a function if it can have different name everywhere? I can change it to default exports but have you ever had issue with finding all occurrences of a function?

Reply:
We've never run into that problem, no. It's part of our style guide that all imports should have the same name as the module being imported.

Reply:
All good, will change it to default export and address all other comments tomorrow morning. 🙂